### PR TITLE
feat: /openstack landing page

### PR DESCRIPTION
**Pull request by mtruj013**
**Note: form will be added in separate PR**

## Done

- Built page as per [doc](https://docs.google.com/document/d/1v1G2y465b_VYVUG9TIV-A_X1dU4kg5Zvrwd2latidCk/edit?tab=t.0) and [design](https://www.figma.com/design/vH19y2J2YqwUfksebSUvTW/OpenStack?node-id=224-7535&node-type=instance&m=dev)

## QA

- View the site locally in your web browser at: https://canonical-com-1394.demos.haus/openstack
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14071

### CODE REVIEW

File: templates/openstack/index.html [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Keep your future in your hands with your own professional cloud</h2>
+      </div>
+      <div class="col">
+        <p>
+          Transform your data centre into a fully functional and professional cloud environment. Take advantage of cloud-based operations, while retaining all the benefits of owning your computing infrastructure.
+        </p>
+        <div class="p-cta-block">
+          <a class="p-button--positive">Get in touch</a>
```
Comment (mtruj013) [inline templates/openstack/index.html:85]: This and any other CTA that is supposed to link to this form will not work for now. See discussion [here](https://warthogs.atlassian.net/browse/WD-14071?focusedCommentId=597407)

**Comment by Sophie-32** [comment]
Found two small typos and marked in the copydoc, you can +1 for me after that

**Comment by mtruj013** [comment]
@Sophie-32 done!

**Comment by lyubomir-popov** [comment]
nearly perfect as usual @mtruj013 

a few small glitches:

- lines in the quote pattern should start from the 4th column on desktop (see in the design)
image

- logos in quotes 0 please check the sizes and update them, they are too large:
image

- please use same-height logos (let me know if the official set is missing any). height of all should be 104px, width depends on individual logos
image

- pls make 2 logos appear on each row on small screens. We can then use this as a basis for the new Logo section with links HOC:
image

**Comment by mtruj013** [comment]
Thanks @lyubomir-popov! I've pushed most of the changes if you could take another look?

For the logos, I sized the Pacific Textiles logo a bit larger as the matching the sizing on the figma left it blurry and kind of unreadable, and the Sicredi logo needs replacing. Also the text on this last logo here looks super tiny compared to the rest, is it possible to update?

image

Lmk if you spot anything else!

**Comment by lyubomir-popov** [comment]
Much better! last couple of things, sorry I missed them the first time:

- p-section shallow around this:

image

- this line should be muted as it is inside a section, not at the top of it:

image

**Review by lizzochek — APPROVED** [review]
LGTM!

**Pull request merged**